src/components/Login/useHandleClickOutside.ts

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useRef } from "react";
+
+export const useOutsideClick = (callback: () => void): React.RefObject<HTMLDivElement> => {
+  const ref = useRef<HTMLDivElement>(null);
```

Comment:
I'm not entirely sure yet how this work, but I don't think this is working as intended since it doesn't take in any ref as props so how does it know which ref to check